examples/experimental/assets_yaml_dsl/assets_yaml_dsl_tests/test_assets_dsl.py

```diff
@@ -69,7 +69,7 @@ def test_execution() -> None:
     assert assets_defs
     assert len(assets_defs) == 1
     assets_def = assets_defs[0]
-    assets_def(context=build_asset_context(), subprocess_resource=SubprocessExecutionResource())
+    assets_def(context=build_asset_context(), subprocess_resource=ExtSubprocess())
```

Comment:
let's rename all `subprocess_resource` instances to `ext_subprocess` or `ext` in a follow up